Dockerfile

```diff
@@ -1,37 +1,31 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
 ARG DEBIAN_FRONTEND=noninteractive
-MAINTAINER https://github.com/huard/raven
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="smith.trevorj@ouranos.ca"
 LABEL Description="Raven WPS" Vendor="Birdhouse" Version="0.18.2"
 
-# Update Debian system
-RUN apt-get update && apt-get install -y \
- build-essential unzip libnetcdf-dev curl \
-&& rm -rf /var/lib/apt/lists/*
-
-# Update conda and mamba
-RUN mamba update -n base conda mamba
+WORKDIR /code
+COPY . /code
 
 # Create conda environment
-COPY environment.yml /opt/wps/
-RUN mamba create --yes -n wps python=3.8 && mamba env update -n wps -f /opt/wps/environment.yml
```

Comment:
Perform all mamba install in the same `RUN` layer for proper clean up at the end.